src/ert/resources/workflows/jobs/internal-gui/scripts/csv_export.py

```diff
@@ -67,6 +70,7 @@ def run(
         # Use the keys (UUIDs as strings) to get ensembles
         ensembles = []
         for ensemble_id in ensemble_data_as_dict:
+            assert self.storage is not None
```

Comment:
(this smells a little bit, can't see how the `storage` function argument is used)

Reply:
Why is `self.storage` used instead of the parameter `storage` ? Is this some sort of backwards compatability thing @oyvindeide ?